### ISSUE TITLE
align in jsdoc

### ISSUE THREAD

**Issue by tcoopman** [Freyskeyd/atom-alignment#13]
```
 * @param {int} xxxxxxxxxxx
 * @param    {string} yyyyyyyyyyyyyyy
 * @param  {function} zzzzzzzzzzz
 * @param  {Object} aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
 * @return {Promise} bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
 */
```

Should align to:

```
 * @param {int}      xxxxxxxxxxx
 * @param {string}   yyyyyyyyyyyyyyy
 * @param {function} zzzzzzzzzzz
 * @param {Object}   aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
 * @return {Promise} bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
 */
```

**Comment by papermoon1978** [comment]
I'm currently working on it ... part of it is solved in 0.11.0. Add the character "{" to the alignment chars and select the 4 lines from the top. Then after ctrl+cmd+a the lines should look like this ...

```
* @param {int} xxxxxxxxxxx
* @param {string} yyyyyyyyyyyyyyy
* @param {function} zzzzzzzzzzz
* @param {Object} aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
* @return {Promise} bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
```

The next step is to correctly align the third part of your blocks using multiple cursors. Currently this doesn't work because it doubles the 3rd part (I missed this during reworking the package, sorry) but I keep you posted.

**Comment by papermoon1978** [comment]
The remaining issue is now solved in the upcoming release (not released yet). Just place some cursors like shown below (the | are marker for the different cursor positions)

```
* @param {int} |xxxxxxxxxxx
* @param {string} |yyyyyyyyyyyyyyy
* @param {function} |zzzzzzzzzzz
* @param {Object} |aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
* @return {Promise} |bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
```

and it will align correctly.
